style.css

```diff
@@ -220,10 +251,10 @@ ul {
   display: flex;
   flex-direction: column;
   align-items: center;
-  margin-top: 50px;
   border-top-left-radius: 100px;
   color: #ebebff;
-  padding: 24px;
+  padding-left: 24px;
+  padding-bottom: 24px;
 }
 
 .contact-me-text {
```

Comment:
- [ ] [Optional] The previous reviewer missed this so I am giving you an optional but kindly implement it. The elements of the form are centered on the page including the `Get in touch button.
- [ ]  [Optional]  Kindly increase the width of the form to make it match Figma's design.
- [ ]  [Optional]  your content is covering whole width but if you see in Figma design the content lies on middle of the page kindly fix it
### your design
![yourrrrr](https://user-images.githubusercontent.com/81727988/178094228-84e8b7d8-5850-420e-b58a-4647e6788723.png)
### Figma design
![your design](https://user-images.githubusercontent.com/81727988/178094239-d119fa7b-65cb-46d9-97b9-57760f5ef435.png)